### DISCUSSION TITLE
Remember Last Location

### DISCUSSION THREAD

**Discussion by Teofilios** [gildas-lormeau/SingleFile#925] [General]
SingleFile does not remember the last download location in Chrome based browsers such as Opera and Brave. I must always navigate each time I want to download a web page using SingleFile. In Firefox, it works fine. It remembers the last location.

**Comment by Teofilios** [comment]
In Options, I have ticked "Save as" dialogue to confirm the file name.

**Comment by Teofilios** [comment]
I also forgot to mention that saving pages as pdf, or images, with the browser's download manager, the last download location is remembered, but not with SingleFile's manager in Opera or Brave. In Firefox, it works fine.

**Comment by gildas-lormeau** [comment]
This is a bug in the browser. There is nothing I can do on my end to fix it in SingleFile. Although, you should be able to circumvent it by disabling the option "Misc > save pages in background" as documented here: https://github.com/gildas-lormeau/SingleFile#known-issues

**Comment by Teofilios** [comment]
Thanks for your reply.

**Reply by gildas-lormeau** [reply]
You're welcome
